### ISSUE TITLE
Links inside the documentation

### ISSUE THREAD

**Issue by merveenoyan** [gradio-app/gradio#675]
It's a feature request, that would be nice if gradio docs had links for headers (like transformers docs attached to this issue), so that when I send a user a component they don't have to search for things. ![image](https://user-images.githubusercontent.com/53175384/154753231-e5765cbc-d9f9-4015-93f5-499ab9d984a4.jpeg)

**Comment by merveenoyan** [comment]
@abidlabs yes, I was looking for a link to highlighted text and couldn't find so I thought it didn't exist across all of the docs (I was on mobile and had no sidebar, then I checked on my PC and there was a sidebar but still it would be nice if we could have these with links as well, I know it's very extra and less priority😅) ![image](https://user-images.githubusercontent.com/53175384/154755464-4191b2b0-9ca2-4692-b04a-ec526775c942.jpeg)

**Comment by abidlabs** [comment]
Gotcha! Well we can probably do something to make this easier. Cc @aliabd
